public/stylesheets/site/2.0/07-interactions.css

```diff
@@ -236,7 +236,7 @@ form li ul.autocomplete li.input {
   border: 0;
 }
 
-.autocomplete .dropdown ul li:hover, li.selected {
```

Comment:
`.autocomplete .dropdown ul li:hover, li.selected` is also present in public skins (Reversi, Snow, Snow Blue) and the skin wizard header styles, do those need to be updated as well?

Reply:
`:sob_blood:` Yes, thanks, I'll do that once I finish Duolingo.

Reply:
Looks good! Do you want to copy the public skins to staging (https://github.com/otwcode/otwarchive/pull/4247#issuecomment-1145315236) before merging?

And looks like we'll have to deal with the [Dreamwidth importing failures](https://otwarchive.atlassian.net/browse/AO3-6347) soon.

Reply:
Yeah, I'll copy the skins over. Can you make an issue for disabling those tests?

Reply:
Skins copied!

Reply:
Filed [AO3-6353](https://otwarchive.atlassian.net/browse/AO3-6353).

Reply:
Will merge after #4249.